### ISSUE TITLE
Download page lacks links for RHEL 8 family

### ISSUE THREAD

**Issue by ergamenes** [rear/rear#2308]
http://relax-and-recover.org/download/ lacks links for Redhat 8 and friends.

**Comment by jsmeix** [comment]
There are no such links because is no Red Hat 8 in
http://download.opensuse.org/repositories/Archiving:/Backup:/Rear/
or
http://download.opensuse.org/repositories/Archiving:/Backup:/Rear:/Snapshot/

I guess this is because ReaR is not built for Red Hat 8 according to
https://build.opensuse.org/repositories/Archiving:Backup:Rear
and this is because it sems there is not yet Red Hat 8 available at
https://build.opensuse.org/project/add_repository_from_default_list/Archiving:Backup:Rear
so currently ReaR cannot be built for Red Hat 8.

But because ReaR is only bash scripts (plus documenatation)
and ReaR's bash scripts are the same for all Linux distributions
it means that, in the end, it does not really matter for which
Linux distribution you use those bash scripts you use.

Bottom line:
I think for Red Hat 8 you can use ReaR that is built for Red Hat 7.
But I am not a Red Hat user so I cannot verify if it is actually true.

FYI: See also
"Version upgrades with Relax-and-Recover" and
"Testing current ReaR upstream GitHub master code" at
https://en.opensuse.org/SDB:Disaster_Recovery
